frontend/src/index.css

```diff
@@ -1,13 +1,98 @@
-body {
+:root {
+  --color-white: #fff;
+  --color-black: #000;
+  --color-light-grey: #f6f6f6;
+  --color-dark: #191919;
+
+  --color-primary: #0000ff;
+  --color-primary-dark: #0000cc;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ul,
+ol,
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
+body {
+  min-height: 100vh;
+  scroll-behavior: smooth;
+  text-rendering: optimizeLegibility;
+  font-family: 'Roboto', sans-serif;
+  line-height: 1.5;
+  font-size: 1.5rem;
+  background-color: var(--color-light-grey);
+  color: var(--color-dark);
+}
+
+a {
+  color: var(--color-primary);
+  text-decoration: none;
+}
+
+a:hover {
+  color: var(--color-primary-dark);
+  text-decoration: underline;
+}
+
+a:not([class]) {
```

Comment:
w styled components nie styleujemy przez klasy albo robimy to naprawde w ostateczności